docs-src/java/testing-activities.md

```diff
@@ -7,7 +7,7 @@ tags:
   - guide-context
 ---
 
-Mocking isolates code undergoing testing so the focus remains on the code, and not on external dependencies or other state. You can test activities using a mocked Activity environment. 
+Mocking isolates code undergoing testing so the focus remains on the code, and not on external dependencies or other state. You can test activities using a mocked Activity environment.
```

Comment:
Looks like this wasn't intended to be a part of the change.
But seems like "mocking isolates code undergoing testing" should either be "Mock isolated code that undergoes testing"
or
"Mocking isolated code, undergoing testing, keeps the focus on the code"

Also, each sentence on a newline.